src/common/Common.ts

```diff
@@ -57,26 +57,28 @@ export class SetUtils {
     }
 }
 
+export enum Platform {
+    desktop = 'desktop',
+    ios = 'ios',
+    android = 'android'
+}
+
 export enum FlagsConfigType {
-    Platform,
+    PlatformType,
```

Comment:
Had to rename this one to avoid clashing with `Platform` enum

Reply:
Can we rename this to `PlatformFlag` instead? My gut check was wondering what the difference between `PlatformType` and `Platform` was, which to me means the names are still too closely related. (I do understand the difference after checking it out. 🙂)